src/components/Book.css

```diff
@@ -1,25 +1,187 @@
+  font-family: 'RobotoSlab', sans-serif;
+  font-size: 1.375rem;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: -0.2px;
+  color: #121212;
+}
+
+.bks_main_cont .books_container .author_tl {
+  font-family: 'RobotoSlab', sans-serif;
+  font-size: 0.875rem;
+  font-weight: 300;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: #4386bf;
+}
+
+.bks_main_cont .container_info {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  margin-top: 4%;
+  margin-bottom: 2%;
+  align-items: center;
+  height: 100px;
+}
+
+.bks_main_cont .books_container .bks_btn {
+  font-family: 'RobotoSlab', sans-serif;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: #4386bf;
+  border: none;
   background-color: #fff;
+  height: 1.2rem;
+}
+
+.bks_main_cont .container_chapter button {
+  border-radius: 3px;
+  width: 80%;
+  background-color: #0290ff;
+  height: 1.8rem;
+  border: none;
+}
+
+.bks_main_cont .books_container .container_info > button:nth-child(1) {
+  height: 1.2rem;
+  border-right: 2px solid #e8e8e8;
+  text-align: left;
+}
+
+.bks_main_cont .books_container .container_info > button:nth-child(2) {
+  border-right: 2px solid #e8e8e8;
+}
+
+.bks_main_cont .perc_container {
+  display: flex;
+  justify-content: center;
+  margin: 0 auto;
+  margin-top: 4%;
+  margin-bottom: 2%;
+  padding-left: 2%;
+  align-items: center;
+  height: 100px;
+  width: 25%;
+}
+
+.bks_main_cont .perc_container .circle_perc {
+  background-image: url(../../public/circles.png);
+  background-repeat: no-repeat, repeat;
+  background-position: center;
+  background-size: 100%;
+  width: 100px;
+  height: 100px;
+}
+
+.bks_main_cont .perc_container .perc_letter {
+  border-right: 2px solid #e8e8e8;
+}
+
+.bks_main_cont .perc_container .perc_letter .pername {
+  color: #121212;
+  font-family: 'Montserrat', sans-serif;
+  font-size: 1.5rem;
+  font-weight: normal;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+}
+
+.bks_main_cont .perc_container .perc_letter .completed_Stat {
+  color: #dbc1c1;
   border-radius: 4px;
-  border: solid 1px #e8e8e8;
+  box-shadow: 0 0 60px 0 rgba(0, 0, 0, 0.05);
+  background-color: #fafafa;
+  font-size: 0.9rem;
 }
 
-.bks_main_cont .books_container .bks_btn {
-  width: 10%;
-  position: relative;
-  right: 45%;
+.bks_main_cont .container_chapter {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  margin: 0 auto;
+  margin-top: 4%;
+  margin-bottom: 2%;
+  padding-left: 2%;
+  height: 100px;
+  width: 30%;
+}
+
+.bks_main_cont .container_chapter h4 {
+  opacity: 0.5;
+  font-family: 'RobotoSlab', sans-serif;
+  font-size: 0.813rem;
+  font-weight: 300;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
```

Comment:
- [ ] The default value for `font-stretch`, `font-style`, `line-height`, `letter-spacing` is "normal", so you don't have to write these CSS properties. To follow the best practice and for Clean CSS code 💯

Reply:
Thank you so much that is very good obervation.